applications/OptimizationApplication/python_scripts/responses/standardized_objective.py

```diff
@@ -0,0 +1,65 @@
+import KratosMultiphysics as Kratos
+import KratosMultiphysics.OptimizationApplication as KratosOA
+from KratosMultiphysics.OptimizationApplication.utilities.optimization_info import OptimizationInfo
+from KratosMultiphysics.OptimizationApplication.utilities.union_utilities import SupportedSensitivityFieldVariableTypes
+from KratosMultiphysics.OptimizationApplication.utilities.communicators.response_function_communicator import ResponseFunctionCommunicator
```

Comment:
I am not fully convinced about this concept/implementation of Communicators like ResponseFunctionCommunicator and OptimizationComponentCommunicator. It is getting like pasta and everyone needs the others. Basically, you are forwarding the implementation and work from this class to another class. Actually, the functions in these Communicators are only utilities and I would highly appreciate if you make them not class rather Python utilities if you do not like repeat function implementation for name, value and sensitivites

Reply:
Moved all the common methods to `StandardizationUtilities`, and removed all the `Communicator` classes.